ts/card-info/Revlog.svelte

```diff
@@ -77,47 +77,97 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
-            <th class="right">{tr2.cardStatsReviewLogTimeTaken()}</th>
-        </tr>
-        {#each revlogRows as row, _index}
-            <tr>
-                <td class="left"><b>{row.date}</b> @ {row.time}</td>
-                <td class="center hidden-xs {row.reviewKindClass}">
-                    {row.reviewKind}
-                </td>
-                <td class="center {row.ratingClass}">{row.rating}</td>
-                <td class="right">{row.interval}</td>
-                <td class="center hidden-xs">{row.ease}</td>
-                <td class="right">{row.takenSecs}</td>
-            </tr>
-        {/each}
-    </table>
+    <div class="revlog-table">
+        <div class="column">
+            <div class="column-head">{tr2.cardStatsReviewLogDate()}</div>
+            <div class="column-content">
+                {#each revlogRows as row, _index}
+                    <div><b>{row.date}</b> @ {row.time}</div>
+                {/each}
+            </div>
+        </div>
+        <div class="column hidden-xs">
+            <div class="column-head">{tr2.cardStatsReviewLogType()}</div>
+            <div class="column-content">
+                {#each revlogRows as row, _index}
+                    <div class={row.reviewKindClass}>
+                        {row.reviewKind}
+                    </div>
+                {/each}
+            </div>
+        </div>
+        <div class="column">
+            <div class="column-head">{tr2.cardStatsReviewLogRating()}</div>
+            <div class="column-content">
+                {#each revlogRows as row, _index}
+                    <div class={row.ratingClass}>{row.rating}</div>
+                {/each}
+            </div>
+        </div>
+        <div class="column">
+            <div class="column-head">{tr2.cardStatsInterval()}</div>
+            <div class="column-content right">
+                {#each revlogRows as row, _index}
+                    <div>{row.interval}</div>
+                {/each}
+            </div>
+        </div>
+        <div class="column hidden-xs">
+            <div class="column-head">{tr2.cardStatsEase()}</div>
+            <div class="column-content">
+                {#each revlogRows as row, _index}
+                    <div>{row.ease}</div>
+                {/each}
+            </div>
+        </div>
+        <div class="column">
+            <div class="column-head">{tr2.cardStatsReviewLogTimeTaken()}</div>
+            <div class="column-content right">
+                {#each revlogRows as row, _index}
+                    <div>{row.takenSecs}</div>
+                {/each}
+            </div>
+        </div>
+    </div>
 {/if}
 
-<style>
-    .left {
-        text-align: start;
+<style lang="scss">
+    .revlog-table {
+        width: 100%;
+        max-width: 50em;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        gap: 0.5em;
```

Comment:
@dae Wasn't `flex-gap` still unsupported on some Anki versions?

Reply:
Good catch, I missed that.

@RumovZ gap is unavailable on our Qt 5.14/5.15 builds, and on older iOS versions:

https://wiki.qt.io/QtWebEngine/ChromiumVersions
https://caniuse.com/flexbox-gap

Here's how a resized window appears on Qt 6:

<img width="432" alt="Screen Shot 2022-01-08 at 9 40 31 am" src="https://user-images.githubusercontent.com/180542/148620826-5db5b266-f130-43c8-9a91-8912dfa172bd.png">

And on Qt 5:

<img width="432" alt="Screen Shot 2022-01-08 at 9 39 29 am" src="https://user-images.githubusercontent.com/180542/148620838-2aaa1047-37df-44fe-bb78-9fa6faff7804.png">

In this case though, I'm not sure it's a big deal - on the desktop, if the user increases the window width a bit, the text will space out, and on mobile we're hiding some of the columns, so the text only gets bunched-up on the narrowest of screens, on devices using old iOS versions. So we can probably get away with it here. :-)

Reply:
Thanks for the info. It's not a big deal to use something else instead, as the new approach is extremely flexible, so I've pushed a fix. 🙂